### PR TITLE
[Feat]: [CCM-8920]: CE Recommenadation index

### PR DESCRIPTION
**Pull request by hitesharinga**
## Describe your changes

1. Added index in ce recommendation timescale table 
2. Exposed on health api in cd ng service, /health checks for db connectivity, This api directly returns healthy 

## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/37062)

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 3 Issues 
* Vulnerability 2 Vulnerabilities 
 * Code issue 1 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/7752dbb3099d2dba3b5d37f5931a5e4c/pull-request/37062)

File: 400-rest/src/main/resources/timescaledb/ce_recommendation_index_query.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+-- Copyright 2021 Harness Inc. All rights reserved.
+-- Use of this source code is governed by the PolyForm Shield 1.0.0 license
+-- that can be found in the licenses directory at the root of this repository, also available at
+-- https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+
+BEGIN;
+
+CREATE INDEX IF NOT EXISTS RECOMMENDATION_ACCOUNTID_LASTUPDATEDAT_ISVALID_RESOURCETYPE_INDEX ON ce_recommendations(accountid, lastprocessedat ,isvalid, resourcetype);
```
Comment (sdsingh07) [inline 400-rest/src/main/resources/timescaledb/ce_recommendation_index_query.sql:8]: Can't the index name be `RECOMMENDATION_ACCOUNTID_LASTPROCESSEDAT_ISVALID_RESOURCETYPE_INDEX`?

**Review by sdsingh07 — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/37062)* status: :shipit: complete! ✓

**Comment by sdsingh07** [comment]
trigger messagecheck

**Pull request merged**